VideoLocker/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -531,37 +523,22 @@ protected void hideSoftKeypad() {
     }
 
     public boolean showErrorMessage(String header, String message) {
-        return showErrorMessage(header, message, true);
-    }
-
-    public boolean showErrorMessage(String header, String message, boolean isPersistent) {
-        LinearLayout error_layout = (LinearLayout) findViewById(R.id.error_layout);
-        if (error_layout == null) {
-            logger.warn("Error Layout not available, so couldn't show flying message");
-            return false;
-        }
-        TextView errorHeader = (TextView) findViewById(R.id.error_header);
-        TextView errorMessageView = (TextView) findViewById(R.id.error_message);
-        if (header == null || header.isEmpty()) {
-            errorHeader.setVisibility(View.GONE);
-        } else {
-            errorHeader.setVisibility(View.VISIBLE);
-            errorHeader.setText(header);
-        }
-        if (message != null) {
-            errorMessageView.setText(message);
-        }
-        ViewAnimationUtil.showMessageBar(error_layout, isPersistent);
+        hideErrorMessage();
```

Comment:
@mdinino This is a bit confusing for me. This method is `showErrorMessage`, why do we call a `hideErrorMessage` as the first line? Is this to make sure there are no dialogs that existed before? If that is the case, is this line an artifact from when we decided to remove the case where an error dialog pops up on no data connection?